docker-compose.yaml

```diff
@@ -55,6 +63,10 @@ services:
       context: ./
       dockerfile: Dockerfile
     restart: unless-stopped
+    deploy:
+      resources:
+        limits:
+          memory: 1G
```

Comment:
This resource restriction was not there before. It is new.

Is there 1 GB more to spare on the server, or do we need to limit the other ones more to have 1 GB for this service?
Actually, I am not sure how much this services needs. 

@joker234